rest/rest-server/src/test/java/functionaltests/RestSchedulerPushPullFileTest.java

```diff
@@ -208,8 +231,10 @@ public void testIt(String spaceName, String spacePath, File destPath, boolean en
         HttpResponse response4 = executeUriRequest(reqDelete);
 
         System.out.println(response4.getStatusLine());
+
         assertHttpStatusOK(response4);
 
-        Assert.assertTrue(destFile + " does not exist", !destFile.exists());
+        Assert.assertTrue(destFile + " still exist", !destFile.exists());
```

Comment:
The message is displayed when the assertion is `false`. So I expect to see a message saying why the assertion was wrong not the reverse of why the assertion is `false`. The javadoc seems not to say a lot about how the message should be written:  

```
     * Asserts that a condition is true. If it isn't it throws an
     * {@link AssertionError} with the given message.
     *
     * @param message the identifying message for the {@link AssertionError} (<code>null</code>
     * okay)
     * @param condition condition to be checked
```